### DISCUSSION TITLE
Pymupdf_addfileannot

### DISCUSSION THREAD

**Discussion by Mano2731995** [pymupdf/PyMuPDF#2059] [Looking for help] [answered]
Hai jorj

I need to change fileannot button color
That default color is yellow(1,1,0)but I don't need border and yellow color I need only pushpin 📌.
how to update that button?

**Comment by JorjMcKie** [comment, marked as answer]
* You can only choose between the symbols made available by MuPDF - see documentation.
* You can change the interior color, but you must use the stroke color for this: annotation symbols have no "fill" color - please again look at the documentation.
* You cannot omit the border lines of the symbol.

**Reply by Mano2731995** [reply]
I got output. Thanks jorj
